### PR TITLE
Issue 4450 fix

### PR DESCRIPTION
**Pull request by ashishdawale20**
This PR should fix issue #4450

### CODE REVIEW

File: src/NUnitFramework/tests/Assertions/ArrayEqualsFailureMessageFixture.cs [hunk 1]
```diff
@@ -118,7 +118,7 @@ public void FiveDimensionedArrays()
             var expectedMessage =
                 "  Expected and actual are both <System.Int32[2,2,2,2,2]>" + NL +
                 "  Values differ at index [0,0,0,1,0]" + NL +
-                TextMessageWriter.Pfx_Expected + "3" + NL +
+                 TextMessageWriter.Pfx_Expected + "3" + NL +
```
Comment (manfred-brands) [inline src/NUnitFramework/tests/Assertions/ArrayEqualsFailureMessageFixture.cs]: I have removed the extra space that shouldn't be there.

**Review by manfred-brands — APPROVED** [review]
Thanks @ashishdawale20 for this PR.
This makes the error message for collections of strings much clearer.

**Comment by stevenaw** [comment]
@OsirisTerje @manfred-brands 
I've noticed a few of the appveyor status checks stalling across PRs. I haven't looked into it myself yet, but is this a known issue or something that needs investigation?

**Comment by manfred-brands** [comment]
I had noticed it as well and had sent @OsirisTerje an email. It prevents me from merging otherwise approved PRs.

**Comment by OsirisTerje** [comment]
We should just remove the appveyor build.  I have been just ignoring them.  You can still merge even if the appveyor build stops.

**Comment by manfred-brands** [comment]
> We should just remove the appveyor build. I have been just ignoring them. You can still merge even if the appveyor build stops.

I can't, it says `Required statuses must pass before merging`

**Comment by OsirisTerje** [comment]
Ok, then we just remove appveyor asap.  Raised issue #4504

**Comment by OsirisTerje** [comment]
I have also removed the appveyor builds from the branch protection rules, so it should work now.

**Pull request merged**